src/com/google/javascript/jscomp/AbstractCommandLineRunner.java

```diff
@@ -2403,7 +2403,7 @@ void printBundleTo(JSChunk module, Appendable out) throws IOException {
       // ES6 modules will need a runtime in a bundle. Skip appending this runtime if there are no
```

Comment:
Just above this is where the PrebuiltAst check runs - we could simplify that as well in this patch to only prebuild the inputs that return true for isEs6Module()?

Reply:
Could you leave this as-is, perhaps with a comment? I agree this is potentially useful, but I'd like to have someone profile some of our internal Closure Compiler users before making any changes.